src/os/ui/layer/labelcontrols.js

```diff
@@ -62,6 +63,12 @@ os.ui.Module.directive('labelcontrols', [os.ui.layer.labelControlsDirective]);
 os.ui.layer.LabelControlsCtrl = function($scope, $element, $timeout) {
   os.ui.layer.LabelControlsCtrl.base(this, 'constructor');
 
+  this['pointGeometry'] = {
```

Comment:
Is this intended for a different PR? Something for setting the label anchor?

Reply:
I'm not sure what that was for either, I'll remove it